bundler/benchmarks/bm_bundle_install_rails.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'support/benchmark_bundler.rb'
+require 'bundler/cli'
+
+# Fixed Nokogiri version because latest version 1.6.8 fails to install
```

Comment:
I haven't tested it with a Docker image. 1.6.8 fails on my local machine with all my Rails projects and there is an open issue on Nokogiri where many are facing problems with it, at last time I checked.